server/controllers/admin.py

```diff
@@ -129,9 +129,8 @@ def get_courses(cid=None):
 @admin.route("/")
 @is_staff()
 def index():
-    courses, current_course = get_courses()
-    if courses and not current_user.is_admin:
-        return redirect(url_for(".course_assignments", cid=courses[0].id))
+    if not current_user.is_admin:
```

Comment:
Admins are a small subset of users that can have super user privileges on ok. We still want instructors and TAs to end up in the either the "instructor" view of their course. Furthermore, the only folks which will be able to access this route are staff because the `@is_staff` decorator is used. 

You should be changing where the `OK` logo links to rather than modifying the `index` controller.

Reply:
@colinschoen Thank you for more information,
Second attempt :)

I modified only `staff/navbar.html` 
users who are not admins will get link to the `student.index` page

Was this a way to solve it or am I still confusing something?